.rubocop.yml

```diff
@@ -0,0 +1,12 @@
+AllCops:
+  Exclude:
+    - 'Rakefile'
+    - 'db/seed.rb'
```

Comment:
Why Rakefile and seeds are excluded?

Usually, we exclude auto-generated files; these two are written by developers.